.github/workflows/pytest.yaml

```diff
@@ -15,16 +15,22 @@ permissions: {}
 
 jobs:
   python-test:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO(b/281098312): pytest fails on windows-latest
+        os: [ubuntu-latest]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
To be honest, i don't think we need all Python versions in between. It kind of makes the matrix bigger and more to navigate. Not opposed  to having it and i presume it's quick and cheap to run, but I've thought that oldest and newest usually should cover most types of errors mostly, i.e. 3.7 and 3.11 in this case.

I'd have Andrew look as well though and it's fine either way i guess.

Reply:
Seems reasonable. Removed the versions in-between.